common/flatpak-transaction.c

```diff
@@ -2998,6 +3102,16 @@ flatpak_transaction_real_run (FlatpakTransaction *self,
   if (!ready_res)
     return flatpak_fail_error (error, FLATPAK_ERROR_ABORTED, _("Aborted by user"));
 
+#ifdef ENABLE_DISKSPACE_CHECK
+  flatpak_transaction_set_enable_diskspace_check (self, TRUE);
```

Comment:
As written, I think this means that passing `--enable-diskspace-check` makes `flatpak_transaction_set_enable_diskspace_check (self, FALSE)` a no-op. That seems strange to me. Why is this a configure flag?

Reply:
hmm, good point. Yeah, keeping two things in sync might be difficult.

I basically wanted to induce disk checking for flatpak command-line too. Hence, the configure time option.

Reply:
Making it a config time parameter was a wrong choice. 
Maybe something like `flatpak install --check-disk-space ... ` ?